pom.xml

```diff
@@ -71,7 +71,7 @@
         <netty.version>4.1.6.Final</netty.version>
         <slf4j.version>1.7.12</slf4j.version>
         <!-- If compiling with different hadoop version also modify default hadoop coordinates in TaskConfig.java -->
-        <hadoop.compile.version>2.3.0</hadoop.compile.version>
```

Comment:
Did you encounter the exception: Exception in thread "main" java.lang.NoSuchMethodError: com.amazonaws.services.s3.transfer.TransferManagerConfiguration.setMultipartUploadThreshold(I)V

in your testing? There's some discussion on it here https://issues.apache.org/jira/browse/HADOOP-12420 and the conclusion at the end is that Hadoop 2.7.3 will only work with aws-java-sdk 1.7.4 until Hadoop 2.8 is released (https://issues.apache.org/jira/browse/HADOOP-12269). I was only able to get the s3a scheme working by downgrading Druid's version of aws-java-sdk to 1.7.4 so I'm curious what your experience was.